docs/src/pages/components/data-grid/group-pivot/group-pivot.md

```diff
@@ -6,6 +6,194 @@ title: Data Grid - Group & Pivot
 
 <p class="description">Use grouping, pivoting, and more to analyze the data in depth.</p>
 
+## Row grouping [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/)
+
+For when you need to group rows based on repeated column values, and/or custom functions.
+
+{{"demo": "pages/components/data-grid/group-pivot/RowGroupingSingleGroupingCol.js", "bg": "inline", "defaultCodeOpen": false}}
```

Comment:
@flaviendelangle Can we have the first demo with a single criteria?
I'd then add a simple description for it like this:
"On the following example, we're grouping all movies based on their production **company**"

Reply:
You are right, done :+1: